src/main/java/duke/save/Storage.java

```diff
@@ -0,0 +1,87 @@
+package duke.save;
+
+import duke.tasks.TaskList;
+import duke.tasks.Task;
+import duke.tasks.Deadline;
+import duke.tasks.Event;
+import duke.tasks.ToDo;
+
+import java.io.*;
```

Comment:
It's recommended to list the imported classes explicity rather than using the wildcard